src/formaction.cpp

```diff
@@ -259,6 +260,26 @@ std::vector<std::string> FormAction::get_suggestions(
 							line);
 					}
 				}
+			} else if (tokens[0] == "exec") {
+				if (tokens.size() <= 2) {
+					std::vector<KeyMapDesc> descs;
+					v->get_keymap()->get_keymap_descriptions(
+						descs,
+						v->get_keymap()->get_flag_from_context(this->id())
+					);
+					std::string start;
+					if (tokens.size() == 2) {
+						start = tokens[1];
+					} else {
+						start = "";
+					}
+					for (KeyMapDesc desc: descs) {
```

Comment:
This needlessly creates copies of descriptions; please use a reference instead: `for (const KeyMapSesc& desc : descs)`.